<?xml version="1.0" ?>
<launch>
    <!-- Load the topic parameters -->
    <rosparam file="$(find butia_quiz)/butia_quiz_config/ros.yaml" command="load"/>
    <!-- Run the princial node -->
    <node name="butia_quiz" pkg="butia_quiz" type="butia_quiz.py"/>
    <!-- Run the quiz node -->
    <node name="butia_quiz_listen" pkg="butia_quiz" type="butia_quiz_listen.py"/>
    <!-- Run the services of butia_speech -->
    <include file="$(find butia_speech)/butia_speech_launchfiles/start_services.launch"/>
</launch>